src/UnderDevCrew/Bundle/Transformer/PlacesModelTransformer.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace UnderDevCrew\Bundle\Transformer;
+
+use Nekland\BaseApi\Transformer\TransformerInterface;
+use UnderDevCrew\Bundle\PlacesApiBundle\Factory\PlacesModelFactory;
+
+class PlacesModelTransformer implements TransformerInterface
+{
+    /**
+     * @var PlacesModelFactory
+     */
+    private $modelFactory;
+
+    /**
+     * @param PlacesModelFactory $placesModelFactory
+     */
+    public function __construct(PlacesModelFactory $placesModelFactory)
+    {
+        $this->modelFactory = $placesModelFactory;
+    }
+
+
+    public function transform($data, $type = self::UNKNOWN)
+    {
+        $jsonData = json_decode($data, true);
```

Comment:
For some readable reasons, can you change your var name as something like $decodedJsonData ? I think this is something witch can render a hard understanding code into a stuff more user friendly. As if I read the code I can stop at the var name to understand what its contend. Of course this is detail.